C/cryptography/Vernam_Cipher.c

```diff
@@ -0,0 +1,179 @@
+/*In Vernam cipher(also known as the One-Time Pad), the length of the plaintext, ciphertext, and key is the same.
```

Comment:
Can you please brief a little about ciphertext in a few lines? That would be helpful.